### PR TITLE
Added Dropdown and "tried" to implement translation

### PR DESCRIPTION
**Pull request by Adarsha-gg**
I created the dropbox it can be edited to add new languages. I also added the preffered_language variable so that the value of the dropbox goes to the class so it can be accessed but I couldn't find a way to actually translate the transcript in real time. This is same as the last pull request I closed with the only change being the whisper's model.

### CODE REVIEW

File: AudioTranscriber.py
Comment (SevaSk) [inline AudioTranscriber.py:18]: minor: space after the comma

File: AudioTranscriber.py
Comment (SevaSk) [inline AudioTranscriber.py:19]: the preffered_language memeber is set but never used

File: AudioTranscriber.py
Comment (SevaSk) [inline AudioTranscriber.py:22]: tiny.en.pt file exists in the root directory but tiny does not exist so this will crash on this line

File: AudioTranscriber.py
Comment (SevaSk) [inline AudioTranscriber.py:89]: We are not passing in the language which we want to transcribe. task="translate"  will translate the speech into English. which is not the flag we want

File: main.py
Comment (SevaSk) [inline main.py:66]: minor: space after comma

File: main.py
Comment (SevaSk) [inline main.py:84]: minor: space after comma

File: AudioTranscriber.py
Comment (SevaSk) [reply AudioTranscriber.py:22]: We also still want to use the tiny.en. model for English because of its better performance.

**Review by SevaSk — CHANGES_REQUESTED** [review]
Thanks for contributing! I cannot merge this PR yet I have left some comment with feedback.

**Comment by Adarsha-gg** [comment]
Thank you for your feedback! I'll try implementing the changes. Also can
you tell me how to actually "translate" it to different languages I
couldn't find it in the whisper's documentation.

**Comment by SevaSk** [comment]
Here is the whisper documentation
https://github.com/openai/whisper/blob/main/README.md

we would want to use the language flag so the line would look something like
self.audio_model.transcribe(file_path, fp16=torch.cuda.is_available(), language= "The ISO 2 Letter Language Codes")

**Comment by Adarsha-gg** [comment]
Hey man I may be wrong but I think as of now whisper can only translate
from different languages to english and not from english to different
languages. I think the purpose of this PR is to perform the latter one
which seems impossible to me or I am just dumb enough to not make it work.

**Comment by SevaSk** [comment]
Whisper can definitely transcribe other languages if you get the feature complete and fully working please feel free to open another pull request! I cant merge this one if the feature is not fully implemented.

**Pull request closed without merging**
